message_center_compassion/models/gmc_message.py

```diff
@@ -333,11 +333,16 @@ def _send_message(self, message_data):
             # Complete failure (messages were not processed)
             fail = onramp_answer.get('content', {
                 'Error': onramp_answer.get('Error', 'None')})
+            results = onramp_answer.get('content', {}).get(
```

Comment:
For some message implementation, error message is on content and not in Error...